### PR TITLE
luci-proto-modemmanager: add luci-proto-modemmanager

### PR DESCRIPTION
**Pull request by nickberry17**
Signed-off-by: Nicholas Smith <nicholas.smith@telcoantennas.com.au>

### CODE REVIEW

**Comment by nickberry17** [comment]
For use when https://github.com/openwrt/packages/pull/9137 is merged

**Comment by nickberry17** [comment]
I think this should actually go into the 18 branch, not master

**Comment by feckert** [comment]
> I think this should actually go into the 18 branch, not master

I don´t think so 18.06 is feature complete.
We could add this to master and then backport this to openwrt-19.07 branch.

**Comment by bobafetthotmail** [comment]
the PR this depends on was merged https://github.com/openwrt/packages/pull/9137
so I guess this can go in too now.

File: protocols/luci-proto-modemmanager/Makefile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
+endef
+
+
+$(eval $(call BuildPackage,luci-proto-modemmanager))
```
Comment (feckert) [inline protocols/luci-proto-modemmanager/Makefile]: Please update this Makefile have a look at https://github.com/openwrt/luci/blob/master/protocols/luci-proto-qmi/Makefile as pattern

**Comment by feckert** [comment]
Sorry but we need to port this to javascript to get merged lua is not working anymore for proto!
We use now in master client side scripting.
All protocols in https://github.com/openwrt/luci/tree/master/protocols have been ported please use this as a pattern

**Pull request closed without merging**